modules/security-jwt/src/main/java/com/devonfw/module/security/jwt/common/impl/JwtManagerImpl.java

```diff
@@ -25,6 +27,7 @@
  *
  * @since 2020.04.001
  */
+@Named
```

Comment:
My vision was that we also create a spring-boot starter.
The `@Named` only helps if the user does a component-scan in this package within its spring-config.
A spring-starter would therefore much better.
Anyhow good that you readded this annotation here. It can only help but not hurt! So thanks for this fix.